<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BingBG</class>
 <widget class="QMainWindow" name="BingBG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bing Backgrounds Getter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Linux Libertine Display O</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="title">
        <string>Get Wallpaper</string>
       </property>
       <widget class="QPushButton" name="btnFetch">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Linux Libertine Display O</family>
          <pointsize>15</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Fetch</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>461</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>Linux Libertine Display O</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QPushButton" name="btnOpenfolder">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>300</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Open in folder</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+O</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpenimg">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>300</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Open image</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+O</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelImg">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>441</width>
          <height>261</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Linux Libertine Display O</family>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width: 1px;
border-style: solid;
border-color: green;</string>
        </property>
        <property name="text">
         <string>No Wallpaper...</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>651</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <family>Linux Libertine Display O</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="title">
        <string>Progress</string>
       </property>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>631</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Linux Libertine Display O</family>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar
{
    text-align:center;
    border-radius:10px;
}
QProgressBar::chunk
{
    background:qlineargradient(spread:pad,x1:0,y1:0,x2:1,y2:0,stop:0 green,stop:1 blue);
    border-radius:10px;
    text-align:center;
}</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="labelCurrent">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>121</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Linux Libertine Display O</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>----</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="labelProg">
        <property name="geometry">
         <rect>
          <x>486</x>
          <y>50</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Linux Libertine Display O</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Bing Backgrounds Getter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="shortcut">
        <string extracomment="Exit the Bing Backgounds Getter">Ctrl+Q</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string extracomment="About">About</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAuthor"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
